src/main/webapp/app/shared/layout/floatButton/FloatButton.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Button } from 'reactstrap';
+import {} from '@fortawesome/free-solid-svg-icons';
```

Comment:
please remove unused import statement

Reply:
Resolved in 9b52ff7a844e085f0b71863cd90d5fa5cd22fb53